src/components/proofs/List.tsx

```diff
@@ -49,8 +49,8 @@ export default function () {
           bottomPadding={savedMessage?.height || 0}
         >
           <div className={innerScrollableBlock}>
-            <EmailProofList />
             <ERC721ProofsList />
+            <EmailProofList />
```

Comment:
They should be under `ERC721` proofs, sorry to include everything in one PR, I am falling asleep 😴